src/gsy_e/gsy_e_core/user_profile_handler.py

```diff
@@ -45,20 +49,20 @@ class ProfileDBConnectionException(Exception):
 
 
 PROFILE_UUID_NAMES = [
```

Comment:
This can be directly imported from gsy-framework and completely removed from here.